keycloak/templates/keycloak.conf.j2

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Keycloak
+After=network.target
+
+[Service]
+Type=idle
+User=keycloak
+ExecStart=/opt/keycloak/bin/kc.sh start-dev
```

Comment:
Shouldn't we use `start` instead of `start-dev` to start the server in production mode instead of dev mode ?
```suggestion
ExecStart=/opt/keycloak/bin/kc.sh start
```

Reply:
en effet, done